src/View/SSViewer_DataPresenter.php

```diff
@@ -139,7 +170,7 @@ public function getInjectedValue($property, $params, $cast = true)
         // Look up the value - either from a callable, or from a directly provided value
         $res = [];
         if (isset($source['callable'])) {
-            $res['value'] = call_user_func_array($source['callable'], $params);
+            $res['value'] = $source['callable'](...$params);
```

Comment:
I was pleasantly surprised by how flexible PHP is with this: https://3v4l.org/I75N8. cc @tractorcow